packages/react-pagination/lib/react-pagination.js

```diff
@@ -1,8 +1,9 @@
 import React, { useState, useEffect } from "react";
 
 export const Pagination = ({ limit, skip, results, skipToFirstLabel, previousLabel, nextLabel, skipToLastLabel, pagesToBottom, ...args }) => {
-	const componentChildren = args.children ? (args.children?.length>1 ? [...args.children] : [args.children]) : [],
-		childElements = [...componentChildren, ...args.child],
+	const componentWrapper = args.children,
```

Comment:
@creador-dev, the pagination component was working fine. Why introduce these changes? Also, this PR is related to documentation, not bug-fixing, improvement or new features; please, submit these changes in a different PR.